store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/utils/TableUtils.java

```diff
@@ -68,22 +68,48 @@ private TableUtils() {
      * Usage:
      * </p>
      * <p>
-     * java -cp hbase-store-[version]-utility.jar uk.gov.gchq.gaffer.hbasestore.utils.TableUtils [pathToSchemaDirectory] [pathToStoreProperties]
+     * java -cp hbase-store-[version]-utility.jar uk.gov.gchq.gaffer.hbasestore.utils.TableUtils [graphId] [pathToSchemaDirectory] [pathToStoreProperties]
      * </p>
      *
-     * @param args [schema directory path] [store properties path]
+     * @param args [graphId] [schema directory path] [store properties path]
      * @throws Exception if the tables fails to be created/updated
      */
     public static void main(final String[] args) throws Exception {
         if (args.length < NUM_REQUIRED_ARGS) {
             System.err.println("Wrong number of arguments. \nUsage: "
-                    + "<schema directory path> <store properties path>");
+                    + "<graphId> <schema directory path> <store properties path>");
             System.exit(1);
         }
 
-        final HBaseStore store = new HBaseStore();
-        store.preInitialise(Schema.fromJson(Paths.get(args[0])),
-                HBaseProperties.loadStoreProperties(args[1]));
+        final StoreProperties storeProps = StoreProperties.loadStoreProperties(args[2]);
```

Comment:
Possibly neater to create private static methods, as in AddUpdateTableIterator to retrieve the args values.  Also makes it a lot easier to read.